static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -301,6 +305,32 @@ class IssueRuleEditor extends AsyncView<Props, State> {
     }
   };
 
+  fetchPreview = async () => {
+    const {organization} = this.props;
+    const {project, rule} = this.state;
```

Comment:
can we cast rule to `UnsavedIssueAlertRule | IssueAlertRule` since there's probably no way its ever null in this function

Reply:
What do you mean by this? Remove the null type in the state?

Reply:
something like this would be okay since we know rule will exist
```ts
const rule = this.state.rule!;
```

or maybe 
```ts
if (!rule) {
	return
}
```